src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ResponseHandlers.scala

```diff
@@ -48,6 +48,21 @@ object ResponseHandlers extends AkkaResponses {
       case Failure(ex)                         => internalServerError(ex, logMessage)
     }
 
+  def createRiskAnalysisResponse[T](logMessage: String)(
+    success: T => Route
+  )(result: Try[T])(implicit contexts: Seq[(String, String)], logger: LoggerTakingImplicit[ContextFieldsToLog]): Route =
+    result match {
+      case Success(s)                             => success(s)
+      case Failure(ex: OperationForbidden.type)   => forbidden(ex, logMessage)
+      case Failure(ex: EServiceNotFound)          => notFound(ex, logMessage)
+      case Failure(ex: TenantNotFound)            => notFound(ex, logMessage)
+      case Failure(ex: TenantKindNotFound)        => notFound(ex, logMessage)
```

Comment:
we can avoid handling these, if they gets raised it means something very bad had happened and it's ok to return 500
```suggestion
```